components/brave_wallet/browser/zcash/zcash_scan_blocks_task.cc

```diff
@@ -87,12 +109,14 @@ void ZCashScanBlocksTask::PrepareScanRanges() {
 
   initial_ranges_count_ =
       std::ceil(static_cast<double>((to - from + 1)) / kBatchSize);
-  scan_ranges_ = std::deque<ScanRange>();
+  pending_scan_ranges_ = std::deque<ScanRange>();
   for (size_t i = 0; i < initial_ranges_count_.value(); i++) {
-    scan_ranges_->push_back(ScanRange{
+    pending_scan_ranges_->push_back(ScanRange{
         static_cast<uint32_t>(from + i * kBatchSize),
         std::min(to, static_cast<uint32_t>(from + (i + 1) * kBatchSize - 1))});
```

Comment:
`initial_ranges_count_` and `i` should be uint32_t

Reply:
I'd strongly suggest to redesign this to use ScanRange as (from, size) instead of current (from, to)